OneCommunityGlobal/HighestGoodNetworkApp
# Alan mark task done dashboard

This PR is to solve a mistake I made when resolving a conflict with development last time I made a pull request. As guided by @Aishwaryak01, I reimplemented her code into the branch. I merged from this branch: https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/685/files.
How to test:
- Make sure that all the functionalities and code changes of PR 685 are in this pull request.
